docs/release-notes/highlights-1.3.1.asciidoc

```diff
@@ -0,0 +1,24 @@
+[[release-highlights-1.3.1]]
+== 1.3.1 release highlights
+
+This is a patch release to address some issues found after the 1.3.0 release. See <<release-highlights-1.3.0>> for the full highlights of the 1.3.x series.
+
+
+
+[float]
+[id="{p}-131-fixes"]
+=== Fixes
+
+[float]
+[id="{p}-131-workaround-kubernetes-bug"]
+==== Workaround rare Kubernetes bug that could result in dataloss
+
+When Secrets created and managed by ECK are copied to a different namespace, they are copied together with a reference to their owner. As the referenced resource is not present in this other namespace, a Kubernetes link:https://github.com/kubernetes/kubernetes/issues/65200[bug] causes garbage collector to remove resources previously referenced, which lead to deletion of all related resources, including PersistentVolumes. This bug is fixed in Kubernetes `1.20`, but as some ECK users might not be able to upgrade when it's released, we are providing the fix on ECK side. Secrets no longer have owner references. To avoid leaking Secrets when custom resources are added and removed, the operator itself is taking care of garbage collecting them.
```

Comment:
```suggestion
ECK manages several Secrets that users may want to copy across namespaces, for being mounted into other applications:
- the Secret containing the `elastic` user password
- the Secret containing the public HTTP certificates to access Elasticsearch, Kibana, Enterprise Search
- the Secret containing the APM Server token
Those Secrets include a reference to their owner resource (for example, the Elasticsearch resource). When a Secret is copied into a different namespace, that owner reference becomes invalid. A Kubernetes link:https://github.com/kubernetes/kubernetes/issues/65200[bug] causes the garbage collection of all resources owned in the original namespace, including PersistentVolumes.
This bug is link:https://github.com/kubernetes/kubernetes/pull/92743[fixed] in Kubernetes `1.20`: resources with an invalid owner references will be automatically deleted, preventing any accidental garbage collection.
However since some ECK users might not be able to upgrade to Kubernetes 1.20 anytime soon, we are providing a fix in ECK directly. All aforementioned Secrets will not have an owner reference set anymore. The ECK operator itself handles garbage collection of those Secrets when appropriate.
```